packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3468,3 +3525,41 @@ class InputDecorationTheme extends Diagnosticable {
     properties.add(DiagnosticsProperty<bool>('alignLabelWithHint', alignLabelWithHint, defaultValue: defaultTheme.alignLabelWithHint));
   }
 }
+
+/// The vertical alignment of text within an input.
+///
+/// A single [y] value that can range from -1.0 to 1.0. -1.0 aligns to the top
+/// of the input so that the top of the first line of text fits within the input
+/// and its padding. 0.0 aligns to the center of the input. 1.0 aligns so that
+/// the bottom of the last line of text aligns with the bottom interior edge of
+/// the input.
+///
+/// See also:
+///
+///  * [TextField.textAlignVertical]
```

Comment:
Links like this are usually followed by a brief explanation. Like:

[TextField.textAlignVertical], which defines the alignment of a text fields prefix, input, and suffix, within the text field.